SharedSource/UnitTests/Development/NUnit/NUnitExtensionTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may not use this
+// file except in compliance with the License.  You may obtain a copy of the
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+
+using System;
+using NUnit.Framework;
+using Remotion.Development.UnitTesting.NUnit;
+namespace Remotion.UnitTests.Development.NUnit
+{
+  [TestFixture]
+  public class NUnitExtensionTests
+  {
+    [Test]
+    public void AssertThatMessageContainsWhitespaces_MessageWithWhitespaces ()
+    {
+      var message = "message with whitespaces";
+      Assert.DoesNotThrow (() => NUnitExtensions.AssertThatMessageContainsWhitespaces (message));
```

Comment:
I can't find another usage of this function in re-motion. I don't think we need to wrap code that should not throw in an assert as the test will fail anyway when an exception is uncaught.